packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -28,15 +41,59 @@ const normalizeContentValue = (value: string) => {
   return value;
 };
 
+/**
+ * Will merge all unconditional css expressions together.
+ *
+ * Input:
+ *
+ * ```
+ * [{ type: 'unconditional', css: 'color: blue' }, { type: 'unconditional', css: 'font-size: 20px' }]
+ * ```
+ *
+ * Output:
+ *
+ * ```
+ * [{ type: 'unconditional', css: 'color: blue; font-size: 20px' }]
+ * ```
+ *
+ * @param arr
+ */
+const reduceUnconditionalCssItems = (arr: Array<CssItem>): Array<CssItem> => {
+  let unconditionalItem: UnconditionalCssItem | undefined;
+
+  return arr.reduce<CssItem[]>((acc, item, index) => {
+    if (!unconditionalItem && item.type === 'unconditional') {
+      unconditionalItem = item;
+    }
+
+    if (index === 0 || item.type !== 'unconditional' || !unconditionalItem) {
```

Comment:
This might fail in below scenario when `conditional` type comes first.
```ts
[{ type: 'conditional', css: 'color: blue' }, { type: 'unconditional', css: 'font-size: 20px' }]
```

Reply:
yeah, i'll rework the function to your suggestion